pages/common/aireplay-ng.md

```diff
@@ -4,6 +4,6 @@
 > Part of `aircrack-ng`.
 > More information: <https://www.aircrack-ng.org/doku.php?id=aireplay-ng>.
 
-- Send a specific number of disassociate packets given an access point's MAC address, a client's MAC address and an interface:
+- Send a specific number of disassociate packets with an access point's MAC address, a client's MAC address and an interface:
```

Comment:
I think this example should be split, because `infinity` is not a `specific number`.

Reply:
Yeah, `infinity` is. But the argument passed is concrete. 😄 `infinity` is a kind of abstraction over the highest possible number accepted by some program.

Reply:
but with `infinity` you can explain how to stop the command with Ctrl+C

Reply:
Sorry, but I didn't get your point: how placeholder relates to interrupting the command? 🤔 Can you provide more details about what u wanna tell me?

Reply:
With a specific number the command exits when the loop completes. with infinity I think the behavior is similar to `ping`, loops until Ctrl+C.

Reply:
In other words u wanna say with any big number this commands is expected to work for a long time? 😄

Reply:
PR was closed, this change was cancelled.